internal/process/steps/gardener_cluster.go

```diff
@@ -53,13 +54,14 @@ func (s *checkGardenerCluster) Run(operation internal.Operation, log logrus.Fiel
 	state := gc.GetState()
 	log.Infof("GardenerCluster state: %s", state)
 	if state != GardenerClusterStateReady {
-		if time.Since(operation.UpdatedAt) > 15*time.Second {
+		// TODO: extract the timeout as a configuration setting, update the operation in the previous step to make the measurement correct way
+		if time.Since(operation.UpdatedAt) > 15*time.Minute {
```

Comment:
this timeout will be extracted as a config in a separate PR (as well as the saving operation to have a measurement)